### PR TITLE
Release 3.1.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Review by Blackbaud-AlexKingman — DISMISSED** [review]

**Review by Blackbaud-DeniseCisneros — DISMISSED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.1.0 (2019-07-22)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Do we need to update the date since we didn't merge this today?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Review by blackbaud-johnly — APPROVED** [review]
LGTM

**Pull request merged**